doc/howto/access_ui.md

```diff
@@ -37,3 +37,18 @@ Complete the following steps to access the LXD web UI:
 )
 
 After setting up the certificates, you can start creating instances, editing profiles, or configuring your server.
+
+## Enable or disable the UI
+
+The {ref}`snap configuration option <howto-snap-configure>` `lxd ui.enable` controls whether the UI is enabled for LXD.
+
+Starting with LXD 5.21, the UI is enabled by default.
```

Comment:
Not sure that's needed to be mentioned here but 5.0 got that backported as well.

Reply:
Hrm did it? That was probably a mistake. You got the commit?